AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenInfo.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Represents token data entity.
+ */
+@interface MSAuthTokenInfo : NSObject <NSCoding>
+
+/**
+ * Auth token string.
+ */
+@property(nonatomic, nullable, copy, readonly) NSString *authToken;
+
+/**
+ * Time and date from which the token began to act.
+ */
+@property(nonatomic, nullable, readonly) NSDate *startTime;
+
+/**
+ * Time and date to which token acted.
+ */
+@property(nonatomic, nullable, readonly) NSDate *endTime;
```

Comment:
Add a comment on the `endTime` property to clarify that in the keychain it's used to store the expiration date.